### ISSUE TITLE
✔️ 🐞 Bug Report: Countdown widget using right time zone in editor but wrong UTC in final page

### ISSUE THREAD

**Issue by gmeligio** [elementor/elementor#17485]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description
<img width="1155" alt="image" src="https://user-images.githubusercontent.com/22875166/149505300-ae7f6639-1070-4677-869b-4eafdf5e993d.png">
<img width="276" alt="image" src="https://user-images.githubusercontent.com/22875166/149506417-c000393c-b7b4-4693-8816-a37e2a052c6f.png">

The Countdown widget is displaying duration in UTC instead of the detected time zone, in my case the time zone is "Europe/Madrid" (UTC+1). I checked that it shows the right duration in the Elementor editor and editor preview but in the preview page and in the final page is in UTC. I tried leaving activated only Elementor and Elementor Pro and I could reproduce it. I tried in new pages and could reproduce it there too.

I expected the duration in the displayed page to be in the right time zone and expected the time zone to match the one in the editor.

## Steps to reproduce

1- Add a Countdown widget.
2- Verify `Date set according to your timezone` is different than UTC.
3- Set date and time in `Due date`.
4- Click Preview to go to the displayed page.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MySQL Community Server (GPL) v5.7.36
	PHP Version: 7.4.25
	PHP Max Input Vars: 5000
	PHP Max Post Size: 64M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.8.3
	Site URL: https://aleasoft.com
	Home URL: https://aleasoft.com
	WP Multisite: No
	Max Upload Size: 64 MB
	Memory limit: -1
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: Europe/Madrid
	Admin Email: tech@aleasoft.com
	Debug Mode: Active

== Theme ==
	Name: Hello Elementor
	Version: 2.4.2
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.71 Safari/537.36

== Active Plugins ==
	Alea Plugin
		Version: 1.1.0
		Author: Alberto

	Broken Link Checker
		Version: 1.11.16
		Author: WPMU DEV

	Elementor
		Version: 3.5.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.5.2
		Author: Elementor.com

	Media Deduper
		Version: 1.5.3
		Author: Cornershop Creative

	Query Monitor
		Version: 3.8.2
		Author: John Blackbourn

	Really Simple SSL
		Version: 5.2.2
		Author: Really Simple Plugins

	Redirection
		Version: 5.1.3
		Author: John Godley

	Site Kit by Google
		Version: 1.48.1
		Author: Google

	Visualizer: Tables and Charts for WordPress
		Version: 3.7.2
		Author: Themeisle

	Wordfence Security
		Version: 7.5.7
		Author: Wordfence

	WP-Sweep
		Version: 1.1.5
		Author: Lester 'GaMerZ' Chan

	WPML Multilingual CMS
		Version: 4.5.2
		Author: OnTheGoSystems

	Yoast Duplicate Post
		Version: 4.3
		Author: Enrico Battocchi & Team Yoast

	Yoast SEO
		Version: 17.9
		Author: Team Yoast

	Yoast SEO Premium
		Version: 17.9
		Author: Team Yoast

== Must-Use Plugins ==
	Elementor Safe Mode
		Version: 1.0.0
		Author: Elementor.com

== Elements Usage ==
	
	page : 2
		button : 3
		divider : 1
		heading : 2
		icon-box : 4
		icon-list : 3
		image : 2
		menu-anchor : 1
		spacer : 6
		text-editor : 8
	section : 12
		button : 2
		google_maps : 2
		heading : 2
		icon : 1
		icon-list : 1
		image : 3
		menu-anchor : 2
		shortcode : 9
		social-icons : 1
		spacer : 8
		text-editor : 2
	wp-page : 497
		button : 80
		countdown : 1
		divider : 40
		form : 6
		heading : 294
		html : 9
		icon : 40
		icon-box : 94
		icon-list : 87
		image : 148
		image-box : 14
		image-gallery : 16
		menu-anchor : 69
		shortcode : 84
		spacer : 290
		tabs : 16
		testimonial : 51
		text-editor : 267

== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Accessibility Improvements: Active
	Import Export Template Kit: Active by default
	Additional Custom Breakpoints: Inactive by default
	Hide native WordPress widgets from search results: Active
	Default to New Theme Builder: Inactive
	Hello Theme Header & Footer: Inactive
	Landing Pages: Active by default
	Color Sampler: Active
	Favorite Widgets: Active
	Admin Top Bar: Active
	Form Submissions: Inactive
	Scroll Snap: Inactive by default

== Log ==
	
Log: showing 20 of 382021-12-15 17:02:37 [info] Elementor/Upgrades - _on_each_version Start  
2021-12-15 17:02:39 [info] Elementor/Upgrades - _on_each_version Finished 
2021-12-15 17:02:39 [info] Elementor/Upgrades - _v_3_5_0_remove_old_elementor_scheme Start  
2021-12-15 17:02:39 [info] Elementor/Upgrades - _v_3_5_0_remove_old_elementor_scheme Finished 
2021-12-15 17:02:39 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.4.8',
  'to' => '3.5.0',
)]
2021-12-15 17:02:54 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.5.1',
  'to' => '3.5.2',
)]
2021-12-15 17:02:54 [info] elementor-pro::elementor_pro_updater Started 
2021-12-15 17:02:54 [info] Elementor Pro/Upgrades - _on_each_version Start  
2021-12-15 17:02:54 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2021-12-15 17:02:54 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.5.1',
  'to' => '3.5.2',
)]
2021-12-20 14:46:55 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.5.0',
  'to' => '3.5.1',
)]
2021-12-20 14:46:55 [info] elementor::elementor_updater Started 
2021-12-20 14:46:55 [info] Elementor/Upgrades - _on_each_version Start  
2021-12-20 14:46:57 [info] Elementor/Upgrades - _on_each_version Finished 
2021-12-20 14:46:57 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.5.0',
  'to' => '3.5.1',
)]
2021-12-20 14:46:59 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.5.0',
  'to' => '3.5.1',
)]
2022-01-07 11:23:58 [info] elementor::elementor_updater Started 
2022-01-07 11:23:58 [info] Elementor/Upgrades - _on_each_version Start  
2022-01-07 11:24:00 [info] Elementor/Upgrades - _on_each_version Finished 
2022-01-07 11:24:00 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.5.1',
  'to' => '3.5.3',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==
</details>

**Comment by serenityinchains** [comment]
I am having the exact same problem.

**Comment by kulterryan** [comment]
I've still having the same issue.

**Comment by LoganStrike** [comment]
@rami-elementor This bug appears to have been re-introduced as of Elementor Pro 3.10. If you roll back to Elementor 3.9.2, the countdown works correctly.

3.9.2 vs 3.10.3:

<img width="1690" alt="Screen Shot 2023-02-04 at 2 32 57 PM" src="https://user-images.githubusercontent.com/16969785/216792281-3e1925d9-b6a1-47dc-93d5-8a14fc9d445e.png">

**Comment by kulterryan** [comment]
Thanks. That worked for me.

**Comment by nicholaszein** [comment]
Hello everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.11.1**! 🥳 

✔️ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
